poc_iot_verifier/migrations/3_poc_report.sql

```diff
@@ -11,6 +11,8 @@ create type reporttype  AS enum (
 
 create table poc_report (
     id bytea primary key not null,
+    -- remote_entropy: allow nulls as only beacon reports will populate this
+    remote_entropy bytea,
```

Comment:
Is this allowed null for backward compat? Reason I ask is that the actual `Report` struct just does `    pub remote_entropy: Vec<u8>,` instead of `pub remote_entropy: Option<Vec<u8>>`

Reply:
the table is populated with both beacon and witness reports but only beacons have the remote_entropy field populated.  For witness reports it defaults to an empty vec.

I looked at having sep tables for beacons and witnesses but it adds a lot of additional overhead and dup code maintaining two sep `impl` and so felt this was a better compromise